### PR TITLE
Bugfix file is updated

### PR DESCRIPTION
**Pull request by Mohammad-Noman-Ahmadi**
some new featues are added

### CODE REVIEW

File: bugfix.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+lsjflkasjfldsf
```
Comment (Noman-Ahmadi) [inline bugfix.txt:1]: this line should be deleted no need of this

**Review by Noman-Ahmadi — CHANGES_REQUESTED** [review]
all looks perfect just do some changes

File: bugfix.txt (hunk 1 above)
Comment (Mohammad-Noman-Ahmadi) [reply bugfix.txt:1]: now its looks perfect please checkit out

**Review by Noman-Ahmadi — APPROVED** [review]
great work now it is perfect

**Pull request merged**